### PR TITLE
Add policy assignments cmdlets

### PR DESCRIPTION
**Pull request by vivsriaus**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vivsriaus__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (visriniv). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/GetAzurePolicyAssignment.cs [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    using System.Management.Automation;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Components;
+    using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Extensions;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Gets the policy assignment.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureRMPolicyAssignment", DefaultParameterSetName = GetAzurePolicyAssignmentCmdlet.ParameterlessSet), OutputType(typeof(PSObject))]
```
Comment (stankovski) [inline src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/GetAzurePolicyAssignment.cs]: Rename to AzureRmXXX

File: src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyAssignment.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Extensions;
+    using Microsoft.Azure.Common.Authentication;
+    using Microsoft.WindowsAzure.Commands.Utilities.Common;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Creates a policy assignment.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRMPolicyAssignment"), OutputType(typeof(PSObject))]
+    public class NewAzurePolicyAssignmentCmdlet : PolicyAssignmentCmdletBase
```
Comment (stankovski) [inline src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyAssignment.cs]: Same as above

File: src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/RemoveAzurePolicyAssignment.cs [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
+{
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Components;
+
+    /// <summary>
+    /// Removes the policy assignment.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "AzureRMPolicyAssignment", DefaultParameterSetName = RemoveAzurePolicyAssignmentCmdlet.PolicyAssignmentNameParameterSet), OutputType(typeof(bool))]
+    public class RemoveAzurePolicyAssignmentCmdlet : PolicyAssignmentCmdletBase
```
Comment (stankovski) [inline src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/RemoveAzurePolicyAssignment.cs]: Same

File: src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/SetAzurePolicyAssignment.cs [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Extensions;
+    using Microsoft.Azure.Common.Authentication;
+    using Microsoft.WindowsAzure.Commands.Utilities.Common;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Sets the policy assignment.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, "AzureRMPolicyAssignment", DefaultParameterSetName = SetAzurePolicyAssignmentCmdlet.PolicyAssignmentNameParameterSet), OutputType(typeof(PSObject))]
+    public class SetAzurePolicyAssignmentCmdlet : PolicyAssignmentCmdletBase
```
Comment (stankovski) [inline src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/SetAzurePolicyAssignment.cs]: Same

**Pull request merged**